src/index.ts

```diff
@@ -15,16 +15,17 @@ async function run() {
 
   await installCli(version, nodePath);
 
-  if (!apiKey) {
-    core.setFailed('Please specify api key as an environment variable');
-    return;
-  }
-
-  if (!(await authenticateWithApiKey(apiKey, nodePath))) {
-    return;
+  if (apiKey) {
+    if (!(await authenticateWithApiKey(apiKey, nodePath))) {
+      return;
+    }
   }
 
   if (workspace) {
+    if (!apiKey) {
```

Comment:
I think it is possible to configure a workspace without an api key or am I wrong?

Reply:
nvm, I think we do.

Reply:
Not sure.  doesn't it try to validate it?